appveyor.yml

```diff
@@ -12,8 +12,8 @@ environment:
 install:
   - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version)
   - npm install yarn -g
-  - yarn install --msvs_version=2013
-  - yarn link
+  - yarn install
+  - yarn link || yarn link webpack
```

Comment:
Same as above, but I don't know for sure if `|| true` works on windows. Unfortunately my VM's are completely broken since the macOS Sierra update so can't test this easily :(.